src/ddoc/macros.d

```diff
@@ -502,8 +502,11 @@ private bool parseAsKeyValuePair(ref Lexer olexer, ref string key, ref string va
 	}
 	else
 		return false;
-	while (!lexer.empty && lexer.front.type == Type.whitespace)
+
+	// parentheses might be used in the key
+	while (!lexer.empty && lexer.front.type.among(Type.whitespace, Type.lParen, Type.rParen, Type.word))
```

Comment:
Ddoc spec says:
> Each line that starts with an identifier followed by an '=' starts a new parameter description.

I'm pretty sure that parens aren't part of identifiers anywhere else in D, so is it really a good idea to make an exception here?